tests/draft2019-09/patternProperties.json

```diff
@@ -147,5 +147,32 @@
                 "valid": true
             }
         ]
+    },
+    {
+        "description": "non-BMP, checks for proper surrogate pair handling for UTF-16",
+        "schema": {
+            "patternProperties": {
```

Comment:
Can you add an invalid case (to this and the previous drafts)?

Seems possible an implementation may ignore these keys entirely and therefore treat every instance as valid.

After that lgtm!

Reply:
Done.